crates/parser-wrapper/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "zombienet-dsl-parser-wrapper"
+name = "dsl-parser-wrapper"
 version = "0.1.6"
```

Comment:
Lets bump the version as well to 0.1.7:
```suggestion
version = "0.1.7"
```

Reply:
Is not really needed, since is already pushed to `npm` as `0.1.6` (firs version of this module).